app/views/external_users/claims/transfer_fee/_fields.html.haml

```diff
@@ -1,42 +1,47 @@
 #transfer-fee
   = f.fields_for :transfer_fee do |tf|
-    .form-section.fx-fee-group.transfer-fee-group.js-block.fx-do-init{data:{"type": "transferFees", autovat: @claim.apply_vat? ? "true" : "false"}}
+    .fx-fee-group.transfer-fee-group.js-block.fx-do-init{ data: { 'type': 'transferFees', autovat: @claim.apply_vat? ? 'true' : 'false' } }
 
       .js-fee-calculator-effector
-        = tf.hidden_field :fee_type_id, value: @claim.transfer_fee.fee_type.id, class: 'js-fee-type'
+        = tf.govuk_text_field :fee_type_id,
+          class: 'js-fee-type',
+          label: { hidden: true },
+          type: 'hidden',
+          value: @claim.transfer_fee.fee_type.id
 
       - unless @claim.transfer_detail.elected_case?
 
         - if @claim.transfer_detail.days_claimable?
           .js-fee-calculator-effector
-            = f.adp_text_field :actual_trial_length,
-              label: t('.actual_trial_length'),
-              input_classes:'form-control form-control-1-4 js-fee-calculator-days',
-              input_type: 'number',
-              errors: @error_presenter
+            = f.govuk_number_field :actual_trial_length,
+              class: 'js-fee-calculator-days',
+              label: { text: t('.actual_trial_length') },
+              min: 0,
+              width: 'one-quarter'
 
         .js-fee-calculator-effector
-          = tf.adp_text_field :quantity,
-            label: t('.quantity'),
-            input_type: 'number',
-            input_classes:'quantity fee-quantity js-fee-calculator-ppe form-control-1-4',
+          = tf.govuk_number_field :quantity,
+            class: 'quantity fee-quantity js-fee-calculator-ppe',
+            label: { text: t('.quantity') },
             value: number_with_precision(tf.object.quantity, precision: 0),
-            errors: @error_presenter,
-            error_key:'transfer_fee.quantity'
+            width: 'one-quarter'
 
       .js-graduated-price-effectee
         .calculated-grad-fee
-        = tf.adp_text_field :amount,
-            label: t('.amount'),
-            input_classes:'total fee-amount form-input-denote__input form-control-1-4 ',
-            input_type: 'currency',
-            value: number_with_precision(tf.object.amount, precision: 2),
-            error_key: 'transfer_fee.amount',
-            errors: @error_presenter
+        = tf.govuk_number_field :amount,
+          class: 'total fee-amount',
+          label: { text: t('.amount') },
+          min: 0,
+          prefix_text: '£',
+          value: number_with_precision(tf.object.amount, precision: 2),
+          width: 'one-quarter'
 
       .js-fee-calculator-success
-        = tf.hidden_field :price_calculated, value: tf.object.price_calculated?
+        = tf.govuk_text_field :price_calculated,
```

Comment:
Parent div with class `.js-fee-calculator-success` can be replaced with the parameter `form_group: { classes: 'js-fee-calculator-success' }`